app/assets/v2/js/pages/dashboard.js

```diff
@@ -290,9 +290,14 @@ var get_search_URI = function(offset) {
 
   var order_by = localStorage['order_by'];
 
-  if (order_by) {
-    uri += '&order_by=' + order_by;
+  if (!document.hackathon) {
+    if (order_by != 'undefined') {
+      uri += '&order_by=' + order_by;
+    }
+  } else {
+    uri += '&event_tag=' + document.hackathon + '&idx_status=open';
```

Comment:
Why are we appending open status here?

Reply:
@SaptakS 
From the issue description and the attached screenshot; I interpreted it to only show open issues in the hackathon case. It this right? @PixelantDesign

Reply:
Anything with the EthSF tag @pinkiebell. Please do not only pull open issues. Thanks!

Reply:
@PixelantDesign
Ok, do we want any separation or specific ordering? Open/Submitted/Closed.